components/support/nimbus-fml/src/workflows.rs

```diff
@@ -205,6 +208,17 @@ mod test {
         Ok(())
     }
 
+    #[test]
+    fn smoke_test_ios_generate() -> Result<()> {
+        generate_and_assert(
```

Comment:
this smoke tests that code generated from our `workflows` can be accessed from our tests. I'll replace this with multiple tests as the code generation progresses